src/platform/testing/e2e/form-tester/index.js

```diff
@@ -97,16 +94,16 @@ const testForm = (testDataSets, testConfig) => {
     }
   });
 
-  Object.keys(testDataSets).forEach(testName =>
+  testDataSets.forEach(({ fileName, contents }) =>
     test(
-      testName,
+      fileName,
       async () => {
         const pageList = await browser.pages();
         const page = pageList[0] || (await browser.newPage());
         await fastForwardAnimations(page);
         await runTest(
           page,
-          getTestData(testDataSets, testName, testConfig.testDataPathPrefix),
+          getTestData(contents, testConfig.testDataPathPrefix),
```

Comment:
Not a huge deal, but can be kind of confusing that the param order is switched up:
`testDataSets.forEach(({ fileName, contents })`
`getTestData(contents, testConfig.testDataPathPrefix),`